lib/task-data/tasks/install-centos.js

```diff
@@ -12,12 +12,12 @@ module.exports = {
         profile: 'install-centos.ipxe',
         hostname: 'localhost',
         comport: 'ttyS0',
-        domain: 'rackhd.github.com',
+        domain: 'rackhd',
```

Comment:
It's fine, just wanted to make sure there was intention behind it. A single word as a domain is uncommon, but as a placeholder/reference example, it's perfectly acceptable